### DISCUSSION TITLE
Deno's 3rd birthday 🎉

### DISCUSSION THREAD

**Discussion by bl-ue** [denoland/deno#10612] [General]
Heads up everyone,

**Deno is turning three years old tomorrow, May 13, 2021 (f7c5e19081920f59ce2006d3255a58fb611b6a17)** 

😉

**Comment by lucacasonato** [comment]
Three years :-)

**Reply by bl-ue** [reply]
Oops, I was thinking of 2020, not 2021 🤣

**Comment by MierenManz** [comment]
In 3 years deno changed alot. I'm really excited to see where deno will be in another year

**Comment by aaronhuggins** [comment]
🎈 🎂 🥳 

I'm most excited by the zero-trust approach to security with Deno, and then by the first-class support for TypeScript. So much has changed, but the things about the project that drew me in are still here.

love it.

**Reply by MierenManz** [reply]
I first really hated that idea of the permission system, but it makes a ton of sense.

**Comment by vicary** [comment]
Hooray!

**Comment by roy2100** [comment]
🎉  🎉  🎉

**Comment by ry** [comment]
Time flies when you're having fun!

We were thinking last year that today might be a good day to release Deno 2.0. But there has been little need for breaking changes, so we're continuing with Deno 1.X indefinitely.
